packages/config/src/options.js

```diff
@@ -369,7 +369,7 @@ export function getNuxtConfig(_options) {
   }
 
   // Add loading screen
-  if (options.dev) {
+  if (options.dev && options.loadingScreen) {
```

Comment:
We already have `options.build.indicator` which can be checked here to not be `false`. Adding another option makes no sense.

Reply:
It seems the build indicator and loading screen are related. With the loading screen disabled I also don't have the build indicator anymore. I was expecting these to be independent features. Can you explain how these are related?

Anyway, turning off the build indicator but leaving the loading screen does currently work. I wouldn't want to make a breaking change.

Reply:
Turns out the build indicator uses the websocket server that loading screen sets up.